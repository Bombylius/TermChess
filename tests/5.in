e2 e4 a7 a6 e4 e5 a6 a5 e5 e6 f7 f6 e6 d7 e8 f7 d7 c8 q
 d7 c8 .q 
