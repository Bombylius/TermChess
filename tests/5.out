17660905521152
16842752
268435456
16777216
1048576
540016640
2048
10240
4
rnQq1bnr/1pp1pkpp/5p2/p7/8/8/PPPP1PPP/RNBQKBNR b KQ - 0 5
